pypeman/msgstore.py

```diff
@@ -484,3 +507,20 @@ async def search(self, start=0, count=10, order_by='timestamp', start_dt=None, e
 
     async def total(self):
         return self._total
+
+    async def _delete_meta_file(self, id):
+        meta_fpath = (Path(self.base_path) / str(id)).with_suffix(".meta")
```

Comment:
just close discussion. but for info.

I try to use _dir or _fname for string objects and _path for path object.
This helps me to know when I have to convert between path objects and strings and what methods to use.

so it would be called `self.base_dir` or `self.base_fname` and you know you have to cast to `Path` but if it's called
`self.base_path` you know, that no casting is required.

Reply:
Yup, i have to clean some repos, but not now, especially if i have to rewrite this class in a near future